File: hw/Vaca/u3/HW30ObserverAndStrategy/HW30Strategy/src/main/java/ec/edu/espe/app/SortApp.java

package ec.edu.espe.app;

import ec.edu.espe.controller.SortController;
import ec.edu.espe.model.SortModel;
import ec.edu.espe.view.SortView;

/**
 *
 * @author MSI-PULSE
 */
public class SortApp {
    public static void main(String[] args) {
        SortView view = new SortView();
        int[] numbers = view.getInputNumbers();

        SortModel model = new SortModel(numbers);
        SortController controller = new SortController(model, view);

        controller.sortNumbers();
    }
}

File: hw/Vaca/u3/HW30ObserverAndStrategy/HW30StrategyJavaScript/app/SortApp.js
import { SortController } from './controller/SortController.js';
import { SortModel } from './model/SortModel.js';
import { SortView } from './view/SortView.js';

class SortApp {
    static main() {
        const view = new SortView();
        const numbers = view.getInputNumbers();

        const model = new SortModel(numbers);
        const controller = new SortController(model, view);

        controller.sortNumbers();
    }
}

SortApp.main();
